### PR TITLE
clarified healing rules and changed to 3 actions

### PR DESCRIPTION
**Pull request by bleehu**
modified:   01_01_Basic_Rules.txt
	modified:   Combat Rules/1_Combat_Rules.txt
	modified:   Items/07_Items.txt

should address issue #673

### CODE REVIEW

**Comment by bleehu** [comment]
I think it's an important balance change to make healing a wound take longer than aiming and firing a gun. It radically changes our action economy. With this significant nerf to med kits and repair kits, we may consider dropping the price for them.

File: 01_01_Basic_Rules.txt [hunk 1]
```diff
@@ -413,34 +413,54 @@ The equations to calculate your bonuses to Saving Throws are as follows:
...
-Using a medic-kit:
-
-	Medic-kit, non-medic user: works as if you are a medic with skill 30.
-
-	Medic-kit, medic user: Works as normal, but add 30 to amount healed.
+    The galaxy is a dangerous place, and you may find in the course of your 
+adventure that you need to heal a wound or even stabalize a friend who is on the
+doorstep of death. The first thing you'll need to know is that treating the 
+injuries of a cybernetic character usually requires the repair skill, while 
+treating a biological character requires the medicine skill. Items such as a
```
Comment (Turtlelord26) [inline 01_01_Basic_Rules.txt]: Gr: An item such as a ...

File: 01_01_Basic_Rules.txt [hunk 2]
```diff
@@ -413,34 +413,54 @@ The equations to calculate your bonuses to Saving Throws are as follows:
...
-	Medic-kit, non-medic user: works as if you are a medic with skill 30.
-
-	Medic-kit, medic user: Works as normal, but add 30 to amount healed.
+    The galaxy is a dangerous place, and you may find in the course of your 
+adventure that you need to heal a wound or even stabalize a friend who is on the
+doorstep of death. The first thing you'll need to know is that treating the 
+injuries of a cybernetic character usually requires the repair skill, while 
+treating a biological character requires the medicine skill. Items such as a 
+medkit or repair kit is required for most characters to be able to treat a wound
+or stabalize a dying character. For stabalizing a dying character, see the
```
Comment (Turtlelord26) [inline 01_01_Basic_Rules.txt]: sp: Stabilize. Several incidences.

File: 01_01_Basic_Rules.txt [hunk 3]
```diff
@@ -413,34 +413,54 @@ The equations to calculate your bonuses to Saving Throws are as follows:
...
+    The galaxy is a dangerous place, and you may find in the course of your 
+adventure that you need to heal a wound or even stabalize a friend who is on the
+doorstep of death. The first thing you'll need to know is that treating the 
+injuries of a cybernetic character usually requires the repair skill, while 
+treating a biological character requires the medicine skill. Items such as a 
+medkit or repair kit is required for most characters to be able to treat a wound
+or stabalize a dying character. For stabalizing a dying character, see the 
+"Stabalizing" section under "Death and Dying" in the Combat Rules.
+
+    Using a medkit or a repair kit to heal a wounded character takes 3 actions
```
Comment (Turtlelord26) [inline 01_01_Basic_Rules.txt]: Order suggestion: Using a ... to heal a wounded character in combat takes 3 Actions.

File: 01_01_Basic_Rules.txt [hunk 4]
```diff
@@ -413,34 +413,54 @@ The equations to calculate your bonuses to Saving Throws are as follows:
...
+in combat. Each healing application heals only one wound at a time. In order for
+any healing to happen, the healer must make a repair or medicine skill check 
+where the DC is the ammount of damage that the wound inflicted. The ammount of
```
Comment (Turtlelord26) [inline 01_01_Basic_Rules.txt]: Sp: Amount. 2 incidences.

File: 01_01_Basic_Rules.txt [hunk 5]
```diff
@@ -413,34 +413,54 @@ The equations to calculate your bonuses to Saving Throws are as follows:
...
+healed hit points is equal to how much the healer overshot the skill check.
+
+    Player Characters of certain classes, like Doctor or Engineer have certain
+abilities that they unlock as they level up that may allow them to heal or 
+repair other characters without a med kit or repair kit.
+
+	For example, Buster gets shot by an SMG for 40 damage and gets shot again by
+a sniper for 80 damage. Buster now has two wounds, the SMG hit and the sniper 
+hit. Buster's friend Sheela uses a medkit to heal the SMG hit. Sheela has a med
+skill of 10, a dex mod of 8 and rolls a 45.
```
Comment (Turtlelord26) [inline 01_01_Basic_Rules.txt]: Clarity point: does the targeted wound have to be declared before the roll, or can it be after? This should be stated explicitly.

File: Combat Rules/1_Combat_Rules.txt [hunk 6]
```diff
@@ -239,15 +239,18 @@ above. Even if you had 5 health and took 80 damage, when you go down you go to 0
...
-of Stabilization. Note that most persistent damage effects, like bleeds or burns, 
-fall off of characters when they are Downed.
+Medicine for primarily organic targets, or Repair for primarily cybernetic 
+characters. If successful, the target is revived and now has 1 HP, is no longer 
+Downed, and may take Actions as normal within the initiative order. A character 
+may be revived in a single encounter a number of times equal to half their Fort, 
+minimum two. Past this limit, stabilized characters are Unconscious in addition 
+to the other effects of Stabilization. Note that most persistent damage effects, 
+like bleeds or burns, fall off of characters when they are Downed. If the 
+stabalizing character starts stabalizing with their last action or two,
```
Comment (Turtlelord26) [inline Combat Rules/1_Combat_Rules.txt:251]: Sp: Stabilize

**Review by Turtlelord26 — CHANGES_REQUESTED** [review]
I am not opposed to testing out slow-heal. I have been thinking about nerfing the amount of each heal, but slowing them down has much the same effect and is certainly worth investigating.
EDIT: Also, the flavor of your way is better. 🐢 

Before greenlighting, I will ask you to additionally apply this nerf to relevant Doc and Engi abilities. I see class balance suffering a lot otherwise.

**Comment by rosebrallier** [comment]
> Before greenlighting, I will ask you to additionally apply this nerf to relevant Doc and Engi abilities. I see class balance suffering a lot otherwise.

Perhaps the Doc and Eng abilities should take 2 actions? 3 would be incredibly punishing to their action economy, and would encompass the idea that they heal faster because they've had lots of practice.

**Review by Turtlelord26 — APPROVED** [review]

**Pull request merged**